waku/v2/node/storage/migration/migrations_scripts/message/00002_addSenderTimeStamp.up.sql

```diff
@@ -26,4 +26,4 @@ INSERT INTO Message (id, receiverTimestamp, contentTopic, pubsubTopic, payload,
     SELECT id, timestamp, contentTopic, pubsubTopic, payload, version, 0
     FROM Message_backup;
 
-DROP TABLE Message_backup;
\ No newline at end of file
+DROP TABLE Message_backup;
```

Comment:
This file does not need to be changed, it is the migration script of an old version of db

Reply:
Indeed, I accidentally removed the line and later restored